manageiq-operator/api/v1alpha1/helpers/miq-components/kafka/kafka.go

```diff
@@ -0,0 +1,548 @@
+					"securityContext": map[string]interface{}{
+						"allowPrivilegeEscalation": false,
+						"capabilities": map[string]interface{}{
+							"drop": []string{"ALL"},
+						},
+						"privileged":             false,
+						"readOnlyRootFilesystem": false,
+						"runAsNonRoot":           true,
+					},
+				},
+				"userOperatorContainer": map[string]interface{}{
+					"securityContext": map[string]interface{}{
+						"allowPrivilegeEscalation": false,
+						"capabilities": map[string]interface{}{
+							"drop": []string{"ALL"},
+						},
+						"privileged":             false,
+						"readOnlyRootFilesystem": false,
+						"runAsNonRoot":           true,
+					},
+				},
+				"tlsSidecarContainer": map[string]interface{}{
+					"securityContext": map[string]interface{}{
+						"allowPrivilegeEscalation": false,
+						"capabilities": map[string]interface{}{
+							"drop": []string{"ALL"},
+						},
+						"privileged":             false,
+						"readOnlyRootFilesystem": false,
+						"runAsNonRoot":           true,
+					},
+				},
+			},
+			"tlsSidecar": map[string]interface{}{
+				"resources": map[string]interface{}{
+					"requests": map[string]interface{}{
+						"cpu":    "500m",
+						"memory": "128Mi",
+					},
+					"limits": map[string]interface{}{
+						"cpu":    "500m",
+						"memory": "128Mi",
+					},
+				},
+			},
+			"userOperator": map[string]interface{}{
+				"resources": map[string]interface{}{
+					"requests": map[string]interface{}{
+						"cpu":    1,
+						"memory": "1Gi",
+					},
+					"limits": map[string]interface{}{
+						"cpu":    1,
+						"memory": "1Gi",
+					},
+				},
+			},
+			"topicOperator": map[string]interface{}{
+				"resources": map[string]interface{}{
+					"requests": map[string]interface{}{
+						"cpu":    1,
+						"memory": "1Gi",
+					},
+					"limits": map[string]interface{}{
+						"cpu":    1,
+						"memory": "1Gi",
+					},
+				},
+			},
+		},
+	}
+}
+
+func KafkaCluster(cr *miqv1alpha1.ManageIQ, client client.Client, scheme *runtime.Scheme) (*unstructured.Unstructured, controllerutil.MutateFn) {
+	kafkaClusterCR := &unstructured.Unstructured{}
+	kafkaClusterCR.SetGroupVersionKind(schema.GroupVersionKind{
+		Group:   "kafka.strimzi.io",
+		Kind:    "Kafka",
+		Version: "v1beta2",
+	})
+	kafkaClusterCR.SetName(cr.Spec.AppName)
+	kafkaClusterCR.SetNamespace(cr.Namespace)
+
+	kafkaCRSpec := KafkaClusterSpec()
+
+	if cr.Spec.StorageClassName != "" {
+		kafkaStorage := kafkaCRSpec["kafka"].(map[string]interface{})["storage"].(map[string]interface{})
+		kafkaStorage["class"] = cr.Spec.StorageClassName
+		zookeeperStorage := kafkaCRSpec["zookeeper"].(map[string]interface{})["storage"].(map[string]interface{})
+		zookeeperStorage["class"] = cr.Spec.StorageClassName
+	}
+
+	kafkaCRSpec = miqutilsv1alpha1.SetKafkaNodeAffinity(kafkaCRSpec, []string{"amd64"})
+
+	kafkaCRSpec = miqutilsv1alpha1.SetKafkaNodeAffinity(kafkaCRSpec, []string{"amd64"})
```

Comment:
```suggestion
	kafkaCRSpec = miqutilsv1alpha1.SetKafkaNodeAffinity(kafkaCRSpec, []string{"amd64", "arm64", "ppc64le", "s390x"})
```

Reply:
@bdunne arm?  It can't hurt to add, but we don't need it right?

Reply:
🤷🏻‍♂️ It's a scheduling limitation.  Since they build it, we may as well allow running there if the resources are available.